protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/PipetteFields/PathField.tsx

```diff
@@ -62,19 +63,23 @@ interface PathButtonProps {
 
 function PathButton(props: PathButtonProps): JSX.Element {
   const { disabled, onClick, id, path, selected, subtitle } = props
-  const [targetProps, tooltipProps] = useHoverTooltip()
+  const [targetProps, tooltipProps] = useHoverTooltip({
+    placement: TOOLTIP_TOP_START,
+  })
   const { t } = useTranslation(['form', 'protocol_steps'])
   // TODO: update the tooltip and images
   const tooltip = (
-    <Tooltip tooltipProps={tooltipProps}>
-      <Box>{t(`step_edit_form.field.path.title.${path}`)}</Box>
-      <img src={PATH_ANIMATION_IMAGES[path]} />
-      <Box>{subtitle}</Box>
+    <Tooltip tooltipProps={tooltipProps} maxWidth={'17rem'}>
+      <Flex gridGap={SPACING.spacing8} flexDirection={DIRECTION_COLUMN}>
+        <Box>{t(`step_edit_form.field.path.title.${path}`)}</Box>
+        <img src={PATH_ANIMATION_IMAGES[path]} width={'100%'} />
```

Comment:
```suggestion
        <img src={PATH_ANIMATION_IMAGES[path]} width="100%" />
```

or if the image size is fixed, you can pass width and height by `px`.

Reply:
I see that the width and height in the Figma design are 361px by 130px, but when I used these dimensions in the code, the image appears much larger than expected.

Reply:
one question for you.
you set 361px x 130px, but Tooltip that is wrapping text and image has maxWidth=272px/17rem?
how did you decide the number?